dotcom-rendering/src/web/components/LinkHeadline.stories.tsx

```diff
@@ -328,9 +328,9 @@ export const DeadBlogSizes = () => (
 				theme: ArticlePillar.News,
 			}}
 			showQuotes={true}
-			kickerText="Large live"
+			kickerText="Large dead"
 			showSlash={true}
-			showPulsingDot={true}
+			showPulsingDot={false}
```

Comment:
Should we turn off the pulsing dot on `DeadBlogs`?

Reply:
I think we should :) Like in the screenshots. 

![Screenshot 2022-02-22 at 10 21 47](https://user-images.githubusercontent.com/19683595/155112730-0f3fb854-f93b-490d-9275-0a770d5bf9f4.png)
![Screenshot 2022-02-22 at 10 21 53](https://user-images.githubusercontent.com/19683595/155112738-936a3ef0-de91-4338-9d82-c396048c5a0b.png)